integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java

```diff
@@ -242,6 +246,10 @@ public int create(String path, @mode_t long mode, FuseFileInfo fi) {
     final int flags = fi.flags.get();
     LOG.trace("create({}, {}) [Alluxio: {}]", path, Integer.toHexString(flags), uri);
 
+    if (uri.getName().length() > MAX_NAME_LENGTH) {
+      LOG.debug("Failed to create {}, file name is too long", path);
```

Comment:
Let us be more explicit here in the error message? e.g., `"Failed to create {}, file name is longer than {} bytes", path, MAX_NAME_LENGTH)`